tests/integration/conftest.py

```diff
@@ -423,3 +423,17 @@ def provider_account(provider_account_params, api):
 @pytest.fixture(scope='module')
 def webhook(api):
     return api.webhooks
+
+
+@pytest.fixture(scope='module')
+def account_plans_params():
+    suffix = get_suffix()
+    name = f"test-{suffix}"
+    return dict(name=name)
+
+
+@pytest.fixture(scope='module')
+def account_plan(account_plans_params, api):
```

Comment:
I think in this case is better to fix account plan
`class AccountPlan(DefaultResource):
    def __init__(self, entity_name='name', **kwargs):  <--- system_name to name
        super().__init__(entity_name=entity_name, **kwargs)`

Reply:
@jsmadis @mganisin What do you think about my proposal?